release/create_release_archives.sh

```diff
@@ -154,9 +175,9 @@ ls -l  "${COMMON_FILES_DIR}/install/kubernetes/helm/istio"
 # Changing dir such that tar and zip files are
 # created with right hiereachy
 pushd "${COMMON_FILES_DIR}/.."
+replace_with_release_charts_url
```

Comment:
Can this be done in https://github.com/istio/istio/blob/master/release/gcb/modify_values.sh instead?

That is where we modify all helm related values.

Reply:
sure. done. Thanks.